Wrappers/Python/cil/optimisation/algorithms/SPDHG.py

```diff
@@ -41,203 +45,422 @@ class SPDHG(Algorithm):
         A convex function with a "simple" proximal
     operator : BlockOperator
         BlockOperator must contain Linear Operators
-    tau : positive float, optional, default=None
-        Step size parameter for Primal problem
-    sigma : list of positive float, optional, default=None
-        List of Step size parameters for Dual problem
-    initial : DataContainer, optional, default=None
+    tau : positive float, optional, default= see note
```

Comment:
I think this should be `None` not `see note`. You can attract attention to the note in the below line.